### ISSUE TITLE
Phenogrid: Add a help tip about interpreting results

### ISSUE THREAD

**Issue by glass-ships** [monarch-initiative/monarch-app#362]
Currently there is no information about how to interpret the phenogrid or what the various scores mean. 

The old version of phenogrid has a question mark which when hovering over gives a pop-up explanation. 

Something similar in the new version would definitely be handy

**Comment by vincerubinetti** [comment]
Related info to capture:

<img width="676" alt="Screenshot 2023-10-04 at 9 50 26 AM" src="https://github.com/monarch-initiative/monarch-app/assets/8326331/fdff68d8-a396-4139-8de0-607b14df3d03">

**Comment by vincerubinetti** [comment]
<img width="402" alt="Screenshot 2023-10-27 at 1 13 26 PM" src="https://github.com/monarch-initiative/monarch-app/assets/8326331/0f338258-9a2f-4e5c-8eb0-8e392055d98c">


Full helper text from UI v2.0:

Phenogrid Faq
How are the similar targets obtained?
We query our owlsim server to obtain the top 100 most-phenotypically similar targets for the selected organism. The grid defaults to showing mouse.
What are the possible targets for comparison?
Currently, the phenogrid is configured to permit comparisons between your query (typically a set of disease-phenotype associations) and one of:
human diseases
mouse genes
zebrafish genes
You can change the target organism by selecting a new organism. The grid will temporarily disappear, and reappear with the new target rendered.
Can I compare the phenotypes to human genes?
No, not yet. But that will be added soon.
Where does the data come from?
The phenotype annotations utilized to compute the phenotypic similarity are drawn from a number of sources:
Human disease-phenotype annotations were obtained from http://human-phenotype-ontology.org/, which contains annotations for approx. 7,500 diseases.
Mouse gene-phenotype annotations were obtained from MGI [(www.informatics.jax.org).](https://previous.monarchinitiative.org/analyze/www.informatics.jax.org) The original annotations were made between genotypes and phenotypes. We then inferred the relationship between gene and phenotype based on the genes that were variant in each genotype. We only perform this inference for those genotypes that contain a single variant gene.
Zebrafish genotype-phenotype annotations were obtained from ZFIN [(www.zfin.org).](https://previous.monarchinitiative.org/analyze/www.zfin.org) The original annotations were made between genotypes and phenotypes, with some of those genotypes created experimentally with the application of morpholino reagents. Like for mouse, we inferred the relationship between gene and phenotype based on the genes that were varied in each genotype. We only perform this inference for those genotypes that contain a single variant gene.
All annotation data, preformatted for use in OWLSim, is available for download from http://code.google.com/p/phenotype-ontologies/
What does the phenogrid show?
The grid depicts the comparison of a set of phenotypes in a query (such as those annotated to a disease or a gene) with one or more phenotypically similar targets. Each row is a phenotype that is annotated to the query (either directly or it is a less-specific phenotype that is inferred), and each column is an annotated target (such as a gene or disease). When a phenotype is shared between the query and target, the intersection is colored based on the selected calculation method (see What do the different calculation methods mean). You can hover over the intersection to get more information about what the original phenotype is of the target, and what is in-common between the two.
Where can I make suggestions for improvements or additions?
Please email your feedback to [info@monarchinitiative.org.](mailto:info@monarchinitiative.org)
What happens to the phenotypes that are not shared?
Phenotypes that were part of your query but are not shared by any of the targets can be seen by clicking the View Unmatched Phenotype link.
Why do I sometimes see two targets that share the same phenotypes have very different overall scores?
This is usually because of some of the phenotypes that are not shared with the query. For example, if the top hit to a query matches each phenotype exactly, and the next hit matches all of them exactly plus it has 10 additional phenotypes that don't match it at all, it is penalized for those phenotypes are not in common, and thus ranks lower on the similarity scale.

**Comment by vincerubinetti** [comment]
What is the score shown at the top of the grid?
The score indicated at the top of each column, below the target label, is the overall similarity score between the query and target. Briefly, for each of the targets (columns) listed, the set of Q(1..n) phenotypes of the query are pairwise compared against all of the T(1..m) phenotypes in the target.

Then, for each pairwise comparison of phenotypes (q x P1...n), the best comparison is retained for each q and summed for all p1..n.

The raw score is then normalized against the maximal possible score, which is the query matching to itself. Therefore, range of scores displayed is 0..100. For more details, please see (Smedley et al, 2012 http://www.ncbi.nlm.nih.gov/pubmed/23660285 and http://www.owlsim.org/)

**Comment by vincerubinetti** [comment]
What do the different calculation methods mean?
For each pairwise comparison of phenotypes from the query (q) and target (t), we can assess their individual similarities in a number of ways. First, we find the phenotype-in-common between each pair (called the lowest common subsumer or LCS). Then, we can leverage the Information Content (IC) of the phenotypes (q,t,lcs) in a variety of combinations to interpret the strength of the similarity.

**Uniqueness reflects how often the phenotype-in-common is annotated to all diseases and genes in the Monarch Initiative knowledgebase. This is simply a reflection of the IC normalized based on the maxIC. IC(PhenotypeInCommon)/maxIC(AllPhenotypes)

**Distance is the euclidian distance between the query, target, and phenotype-in-common, computed using IC scores.
d=(IC(q)-IC(lcs))2+(IC(t)-IC(lcs))2

This is normalized based on the maximal distance possible, which would be between two rarely annotated leaf nodes that only have the root node (phenotypic abnormality) in common. So what is depicted in the grid is 1-dmax(d)

**Ratio(q) is the proportion of shared information between a query phenotype and the phenotype-in-common with the target.
ratio(q)=IC(lcs)/IC(q)*100

**Ratio(t) is the proportion of shared information between the target phenotype and the phenotype-in-common with the query.
ratio(t)=IC(lcs)/IC(t)*100

**Comment by vincerubinetti** [comment]
What are the different ways that phenotypes can be sorted?
The phenotypes that are shown on the left side of the grid may be sorted using one of three methods. More options may be available in the future.
Alphabetical - A-Z
Frequency and Rarity - Phenotypes are sorted by the sum of the phenotype values across all models/genes
Frequency (Default) - Phenotypes are sorted by the count of the number of model/gene matches per phenotype

**Comment by kevinschaper** [comment]
I believe that we addressed this with the info, and links out to documentation on the scoring mechanisms.
